src/replaykit.cs

```diff
@@ -203,9 +204,12 @@ interface RPBroadcastConfiguration : NSCoding, NSSecureCoding {
 		[Export ("clipDuration")]
 		double ClipDuration { get; set; }
 
-		// TODO review, once API documentation is available, if we can provide an easier to use strongdictionary
 		[NullAllowed, Export ("videoCompressionProperties", ArgumentSemantic.Strong)]
-		NSDictionary<NSNumber, INSSecureCoding> VideoCompressionProperties { get; set; }
+#if XAMCORE_2_0
```

Comment:
I've removed the XAMCORE_2_0 ifdef now, it was a leftover from tracking down a build problem.